### ISSUE TITLE
ACCESSIBILITY MOBILE VIEW

### ISSUE THREAD

**Issue by moodle0909** [moodle-an-hochschulen/moodle-theme_boost_union#1052]
<img width="710" height="432" alt="Image" src="https://github.com/user-attachments/assets/c27c2a42-2836-4f09-add6-36541694c81a" />

People accessing web applications with the keyboard should be able to recognize which element has been activated with the keyboard focus. To achieve this, the retention of focus must be clearly marked.

As is well known, the highlighted control element is not highlighted when activated. For people with motor impairments who work manually with the keyboard, the focus position is not recognizable, making orientation difficult.
